src/playbook.yaml

```diff
@@ -10,38 +75,37 @@
     - name: Add R-Project apt repository
       shell: "aptdcon --add-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu {{ ansible_distribution_release }}-cran40/' && aptdcon --refresh"
 
-    - name: install packages
-      shell: "yes | aptdcon --hide-terminal --install 'zlib1g-dev libclang-dev libssl-dev libffi-dev libcurl4-openssl-dev libapparmor1 libssl1.0.0 r-base r-base-dev libxml2-dev apache2 apache2-dev ssl-cert flex python3-boto3 libpq5'"
+    # Install essential R packages
```

Comment:
installing packages from CRAN is slow and it can also be a little flaky.  Moving it up so that build failures will happen sooner rather than later.